components/pdfviewer/annotations.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Annotations
+page_title: PdfViewer - Annotations
+description: Annotations in the PDF Viewer for Blazor. How to add and configure annotations in the PDF Viewer for Blazor
+slug: pdfviewer-annotations
+tags: telerik,blazor,pdf,pdfviewer,annotations
+published: True
+position: 15
+---
+
+# PdfViewer Annotations
+
+The PdfViewer supports a set of annotation types to enhance PDF documents. For example, users can highlight important text areas or add notes as free text.
+
+The PDFViewer creates a dedicated layer for the annotations and includes them when downloading or printing the file. If the user opens a file with annotations, they can edit only the annotations that were added using the PDFViewer `Annotations` tool. The PDFViewer does not support editing annotations added in a different PDF editor.
+
+To add and manage the annotations, the user can click on the dedicated button in the [PDF Viewer Toolbar]({%slug pdfviewer-toolbar%}). It opens an annotations bar with the available annotation types.
+
+## Supported Annotations
+
+The PDFViewer provides the following annotation types:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Annotation Type | Description |
+| --- | --- |
+| **Text Highlight** | Allows highlighting the selected text in the document. Users can choose the desired highlight color. |
+| **Free Text** | Allows creating an element with free text anywhere in the document. Provides option to select the desired text color and font size. |
+
+## Working with Annotations
+
+**To manage the annotations the user should:**
+
+1. Click on the `Annotations` tool in the [Toolbar]({%slug pdfviewer-toolbar%}) to open the Annotations bar with the available annotation types.
+1. Click the desired annotation type to toggle it. You need to first "turn on" the annotation tool to add or edit annotations of the corresponding type. If none of the annotation tools are enabled, the PDFViewer will not allow interacting with the document to create or edit annotations.
+
+**From this point the user can:**
+
+1. **Create an annotation** - for that purpose the user can select a non-annotated text to highlight it or click on an area that does not have free text element to create one. Once they create the annotation, a popup with the available customization options will appear.
```

Comment:
Use the `&mdash;` [here](https://docs.telerik.com/style-guide/punctuation#lists) and throughout the list. Note the capitalization after the em dash. 

```suggestion
1. **Create an annotation**&mdash;The user can select a non-annotated text to highlight it or they can click an area that does not have free text element to create an annotation. Once they create the annotation, a popup with the available customization options will appear.
```